### PR TITLE
Aboutus style #10

### LINKED ISSUE #10: About us
#1
as a user i want to knew more about elegant website 

**Acceptance Criteria**

- [ ] display about us information from database
- [ ] Styling the page

### PR DESCRIPTION
**Pull request by walaamedhat**
- fixes #10  

- related to #10   

- make aboutus page html,css

### CODE REVIEW

File: public/css/login.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
+body {
```
Comment (muhmushtaha) [inline public/css/login.css:1]: we have a branch for login which has the login files, why we have login files here ?

File: public/aboutus.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    <div class="burgerButton">
+        <div class="slice"></div>
+        <div class="slice"></div>
+        <div class="slice"></div>
+      </div>
+    </div>
+
+    <div class="container" id="container">
+      <h1>About Us</h1>
+      <center><div class="parag"><p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem.</p>
```
Comment (muhmushtaha) [inline public/aboutus.html:32]: we shouldn't use center tag anymore!
and in the small screen the text should be wrapped

**Review by muhmushtaha — CHANGES_REQUESTED** [review]

File: public/aboutus.html [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    <header>
+      <form id="headerForm" class="headerForm" action="" method="post">
+        <input type="submit" name="" value="Login">
+        <input type="submit" name="" value="Sign Up">
+      </form>
+    </header>
+      <div class="burgerAndLogo" id="burgerAndLogo">
+        <div></div>
+    <div><img src="./imgs/logo.png" alt="logo" class="logo"></div>
+    <div class="burgerButton">
```
Comment (Salam-Dalloul) [inline public/aboutus.html:23]: i think it would be better if you use font awesome for this :smile:

File: public/css/aboutus.css [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+  background-color: #9e7945;
+  margin-bottom: 5px;
+}
+/* end burgerButton */
+
+.burgerAndLogo {
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: center;
+  justify-content: space-around;
```
Comment (Salam-Dalloul) [inline public/css/aboutus.css:62]: you have to use one property for justify content :smile_cat:

File: public/css/aboutus.css [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+.column img:hover {
+  box-shadow: 0 0 2px 1px rgba(0, 140, 186, 0.5);
+  border-radius: 50%;
+  -webkit-transform: scale(0.9);
+  -ms-transform: scale(0.9);
+  transform: scale(0.9);
+  opacity: 0.7;
+}
+
+footer {
```
Comment (Salam-Dalloul) [inline public/css/aboutus.css:105]: you should use classes, it would be better :D

**Review by Salam-Dalloul — CHANGES_REQUESTED** [review]
check these :smiling_imp:

**Pull request closed without merging**